internals/secrethub/service_gcp_link.go

```diff
@@ -0,0 +1,193 @@
+package secrethub
+
+import (
+	"fmt"
+	"os/exec"
+	"runtime"
+	"time"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/progress"
+	"github.com/secrethub/secrethub-cli/internals/cli/ui"
+	"github.com/secrethub/secrethub-cli/internals/secrethub/command"
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+)
+
+// ServiceGCPLinkCommand create a new link between a SecretHub namespace and a GCP project.
+type ServiceGCPLinkCommand struct {
+	namespace string
+	projectID string
+	io        ui.IO
+	newClient newClientFunc
+}
+
+// NewServiceGCPLinkCommand creates a new ServiceGCPLinkCommand.
+func NewServiceGCPLinkCommand(io ui.IO, newClient newClientFunc) *ServiceGCPLinkCommand {
+	return &ServiceGCPLinkCommand{
+		io:        io,
+		newClient: newClient,
+	}
+}
+
+func (cmd *ServiceGCPLinkCommand) Run() error {
+	client, err := cmd.newClient()
+	if err != nil {
+		return err
+	}
+
+	_, err = client.IDPLinks().GCP().Get(cmd.namespace, cmd.projectID)
+	if !api.IsErrNotFound(err) {
```

Comment:
How would you feel about adding an `Exists` func to `client.IDPLinks().GCP()` which does exactly this?

Reply:
This could also be used in the delete command.

Reply:
:check